pkg/webhooks/server.go

```diff
@@ -73,6 +81,7 @@ func NewServer(
 	configuration config.Configuration,
 	metricsConfig metrics.MetricsConfigManager,
 	debugModeOpts DebugModeOptions,
+	polexOpts ExceptionOptions,
```

Comment:
Why do we need to pass that to the server ? Only the handler cares about that right ?